catroid/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -1758,7 +1758,7 @@ precisa de acesso de leitura e escrita a ele. Você sempre pode alterar as permi
     <string name="formula_editor_logic_comparison">Operadores de comparação</string>
     <string name="formula_editor_object_general">Propriedades gerais</string>
     <string name="formula_editor_object_movement">Propriedades de movimento</string>
-    <string name="formula_editor_device_visual_detection">Sensores visuais</string>
+    <string name="formula_editor_device_face_detection">Sensores visuais</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations